### ISSUE TITLE
connections pane images missing in version 1.4.1

### ISSUE THREAD

**Issue by dskard** [r-dbi/odbc#711]
### Issue Description and Expected Result
Did we lose some images in the connections pane between version 1.4.0 and 1.4.1?

In 1.4.0, I see images next to the schema that allow us to explore the Tables in the Schemas:
![odbc_1 4 1_missing_toggle_image_20231223_2](https://github.com/r-dbi/odbc/assets/8673892/dd875e61-8c34-4615-b4c4-0fd71edcb9b6)

In 1.4.1, I don't see the images next to the schema:
![odbc_1 4 1_missing_toggle_image_20231223_3](https://github.com/r-dbi/odbc/assets/8673892/f884f898-159a-4863-9431-f0884c193434)

### Database
Databricks Spark

### Reproducible Example

<details>
<summary>Session Info</summary>

```r
> devtools::session_info()
─ Session info ─────────────────────────────────────────────────────────────────────
 setting  value
 version  R version 4.1.3 (2022-03-10)
 os       Ubuntu 22.04.3 LTS
 system   x86_64, linux-gnu
 ui       RStudio
 language (EN)
 collate  C.UTF-8
 ctype    C.UTF-8
 tz       Etc/UTC
 date     2023-12-24
 rstudio  2024.04.0-daily+113.pro1 Chocolate Cosmos (server)
 pandoc   NA

─ Packages ─────────────────────────────────────────────────────────────────────────
 package     * version date (UTC) lib source
 bit           4.0.5   2022-11-15 [2] RSPM (R 4.1.0)
 bit64         4.0.5   2020-08-30 [2] RSPM (R 4.1.0)
 blob          1.2.4   2023-03-17 [2] RSPM (R 4.1.0)
 cachem        1.0.8   2023-05-01 [2] RSPM (R 4.1.0)
 cli           3.6.2   2023-12-11 [2] RSPM (R 4.1.0)
 DBI         * 1.2.0   2023-12-21 [2] RSPM (R 4.1.0)
 devtools      2.4.5   2022-10-11 [2] RSPM (R 4.1.0)
 digest        0.6.33  2023-07-07 [2] RSPM (R 4.1.0)
 ellipsis      0.3.2   2021-04-29 [2] RSPM (R 4.1.0)
 fastmap       1.1.1   2023-02-24 [2] RSPM (R 4.1.0)
 fs            1.6.3   2023-07-20 [2] RSPM (R 4.1.0)
 glue          1.6.2   2022-02-24 [2] RSPM (R 4.1.0)
 hms           1.1.3   2023-03-21 [2] RSPM (R 4.1.0)
 htmltools     0.5.7   2023-11-03 [2] RSPM (R 4.1.0)
 htmlwidgets   1.6.4   2023-12-06 [2] RSPM (R 4.1.0)
 httpuv        1.6.13  2023-12-06 [2] RSPM (R 4.1.0)
 later         1.3.2   2023-12-06 [2] RSPM (R 4.1.0)
 lifecycle     1.0.4   2023-11-07 [2] RSPM (R 4.1.0)
 magrittr      2.0.3   2022-03-30 [2] RSPM (R 4.1.0)
 memoise       2.0.1   2021-11-26 [2] RSPM (R 4.1.0)
 mime          0.12    2021-09-28 [2] RSPM (R 4.1.0)
 miniUI        0.1.1.1 2018-05-18 [2] RSPM (R 4.1.0)
 odbc          1.4.1   2023-12-21 [1] RSPM (R 4.1.0)
 pkgbuild      1.4.3   2023-12-10 [2] RSPM (R 4.1.0)
 pkgconfig     2.0.3   2019-09-22 [2] RSPM (R 4.1.0)
 pkgload       1.3.3   2023-09-22 [2] RSPM (R 4.1.0)
 profvis       0.3.8   2023-05-02 [2] RSPM (R 4.1.0)
 promises      1.2.1   2023-08-10 [2] RSPM (R 4.1.0)
 purrr         1.0.2   2023-08-10 [2] RSPM (R 4.1.0)
 R6            2.5.1   2021-08-19 [2] RSPM (R 4.1.0)
 Rcpp          1.0.11  2023-07-06 [2] RSPM (R 4.1.0)
 remotes       2.4.2.1 2023-07-18 [1] RSPM (R 4.1.0)
 rlang         1.1.2   2023-11-04 [2] RSPM (R 4.1.0)
 rstudioapi    0.15.0  2023-07-07 [2] RSPM (R 4.1.0)
 sessioninfo   1.2.2   2021-12-06 [2] RSPM (R 4.1.0)
 shiny         1.8.0   2023-11-17 [2] RSPM (R 4.1.0)
 stringi       1.8.3   2023-12-11 [2] RSPM (R 4.1.0)
 stringr       1.5.1   2023-11-14 [2] RSPM (R 4.1.0)
 urlchecker    1.0.1   2021-11-30 [2] RSPM (R 4.1.0)
 usethis       2.2.2   2023-07-06 [2] RSPM (R 4.1.0)
 vctrs         0.6.5   2023-12-01 [2] RSPM (R 4.1.0)
 xtable        1.8-4   2019-04-21 [2] RSPM (R 4.1.0)

 [1] /home/x/R/x86_64-pc-linux-gnu-library/4.1
 [2] /opt/R/4.1.3/lib/R/library

────────────────────────────────────────────────────────────────────────────────────
```

Launch an RStudio Pro session.

Setup your .Renviron file with the following variables:
```r
DATABRICKS_HOST=...
DATABRICKS_TOKEN=...
DATABRICKS_HTTPPATH=...
```

Install the odbc package, version 1.4.1
```r
install.packages("remotes")
remotes::install_version("odbc",version="1.4.1")
```

You may need to restart the R session to get the package and .Renviron to be picked up.

Start a connection to a Databricks resource:
```r
library(DBI)
packageVersion("odbc")
con <- DBI::dbConnect(
  odbc::databricks(),
  HTTPPath = Sys.getenv("DATABRICKS_HTTPPATH")
)
```

After the connection has been established, expand one of the Catalogs (`samples` in the example above) to expose the Schemas (`default`, `nyctaxi`, `tpch` in the example above). Check if the Schemas have an image next to them that is clickable with your mouse.

</details>

**Comment by detule** [comment]
Thanks for the report!

Do you have the ability to test and see if the situation improves with https://github.com/r-dbi/odbc/pull/715

Thanks again

**Comment by dskard** [comment]
Yes, the images show up again with the fix in #715

**Comment by dskard** [comment]
![odbc_1 4 1_pr_715_missing_toggle_image_20240103_1](https://github.com/r-dbi/odbc/assets/8673892/1316bddd-e583-4967-a796-2a8399f72ac9)
